Signal/src/ViewControllers/ConversationView/Cells/OWSSystemMessageCell.m

```diff
@@ -169,6 +169,9 @@ - (void)applyTitleForInteraction:(TSInteraction *)interaction
     OWSAssert(interaction);
     OWSAssert(label);
 
+    // Update cell to reflect changes in dynamic text.
+    self.titleLabel.font = [self titleFont];
```

Comment:
We need to update all cell's fonts in their "configure" method in case dynamic type sizing has changed since this cell was created.

Reply:
👍 Setting label fonts can be (just a little) expensive, so we might want to set only if not already equal.